src/offline/mod_offline_api.erl

```diff
@@ -2,33 +2,33 @@
 
 -export([delete_expired_messages/1, delete_old_messages/2]).
 
--spec delete_expired_messages(jid:lserver()) ->
-    {ok | domain_not_found | server_error, iolist()}.
+-type(api_result()) :: {ok, string()} | {domain_not_found | server_error, iolist()}.
```

Comment:
`io_lib:format` as used e.g. in line 26 does not return a string - it returns an iolist instead.
```suggestion
-type(api_result()) :: {ok | domain_not_found | server_error, iolist()}.
```